File: Java/mediation-bidder/src/main/java/vdo/ai/sample/java/mediation/bidder/MainActivity.java
package vdo.ai.sample.java.mediation.bidder;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import vdo.ai.sample.java.mediation.bidder.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.adManager.setOnClickListener(v -> {
            startActivity(new Intent(this, AdManagerHomeActivity.class));
        });
        binding.admob.setOnClickListener(v ->{
            startActivity(new Intent(this, AdMobHomeActivity.class));
        });
    }
}

File: Kotlin/example-mediation-bidder/src/main/java/vdo/ai/android/example/mediation/bidder/MainActivity.kt
package vdo.ai.android.example.mediation.bidder

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import vdo.ai.android.example.mediation.bidder.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    var binding: ActivityMainBinding? = null
    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding?.root)
        binding?.adManager?.setOnClickListener {
            startActivity(
                Intent(
                    this,
                    AdManagerHomeActivity::class.java
                )
            )
        }
        binding?.admob?.setOnClickListener {
            startActivity(
                Intent(
                    this,
                    AdMobHomeActivity::class.java
                )
            )
        }
    }
}